lib/ex_doc/formatter/html.ex

```diff
@@ -82,19 +88,45 @@ defmodule ExDoc.Formatter.HTML do
     end
   end
 
-  defp generate_readme(output, module_nodes, config, modules, exceptions, protocols) do
+  defp generate_readme(output, module_nodes, config, modules, exceptions, protocols, logo_path) do
     readme_path = Path.expand(config.readme)
-    write_readme(output, File.read(readme_path), module_nodes, config, modules, exceptions, protocols)
+    write_readme(output, File.read(readme_path), module_nodes, config, modules, exceptions, protocols, logo_path)
   end
 
-  defp write_readme(output, {:ok, content}, module_nodes, config, modules, exceptions, protocols) do
+  defp process_logo_metadata(config) do
```

Comment:
@josevalim  I just though that it's better to read a few chars (16) of the file and then validate the signature. But, I can change this.